pages/linux/ffuf.md

```diff
@@ -0,0 +1,28 @@
+# ffuf
+
+> Subdomain and directory discovery tool.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Discover directories using a [w]ordlist on a target [u]rl with [c]olorized and [v]erbose ouput:
```

Comment:
I didn't mean that there was any error with the letters for the flags being encased. I just personally dislike them, however if everyone else thinks they are fine then feel free to merge

Reply:
I'm also not a fan, but I think that should be discussd in a dedicated issue.

Reply:
@CleanMachine1 the PR got too long, the best was to merge it, and now we can refine the page.